edgelet/edgelet-utils/src/ser_de.rs

```diff
@@ -67,6 +68,17 @@ where
         .context(ErrorKind::SerdeClone)?)
 }
 
+pub fn serialize_ordered<S>(
+    x: &HashMap<String, String>,
+    serializer: S,
+) -> StdResult<S::Ok, S::Error>
+where
+    S: serde::Serializer,
```

Comment:
clippy is pinned to an older version that doesn't add extern crates to the prelude, so you'll need to change this to `S: ::serde::Serializer` or `use serde::Serializer;` `S: Serializer`